src/test/groovy/graphql/execution/reactive/CompletionStageMappingPublisherTest.groovy

```diff
@@ -38,11 +47,17 @@ class CompletionStageMappingPublisherTest extends Specification {
 
         when:
         Publisher<Integer> rxIntegers = Flowable.range(0, 10)
+        Executor executor = CompletableFuture.delayedExecutor(50, TimeUnit.MILLISECONDS)
```

Comment:
The tests are failing because `CompletableFuture.delayedExecutor` is a Java 9 API